docs/deploy.md

```diff
@@ -119,39 +94,21 @@ cf-for-k8s can be configured to [use an external database](platform_operators/ex
 
       ```
 
-      1. Update `<my_username>` with your docker username.
-      1. Update `<my_password>` with your docker password.
-
-   1. To configure a Google Container Registry, add the following registry config block to the end of `cf-values.yml` file.
-
-      ```yml
-      app_registry:
-        hostname: gcr.io
-        repository_prefix: gcr.io/<gcp_project_id>/cf-workloads
-        username: _json_key
-        password: |
-          <contents_of_service_account_json>
-      ```
-
-      1. Update the `gcp_project_id` portion to your GCP Project Id.
-      1. Change `contents_of_service_account_json` to be the entire contents of your GCP Service Account JSON.
+      Update `<my_username>` and `<my_password>` with your docker username and password that you created in the above section [Setup docker registry](#setup-a-docker-registry).
 
 1. Run the following commands to install Cloud Foundry on your Kubernetes cluster.
 
       i. Render the final K8s template to raw K8s configuration
       ```console
       ytt -f config -f ${TMP_DIR}/cf-values.yml > ${TMP_DIR}/cf-for-k8s-rendered.yml
       ```
-      > cf-for-k8s uses [ytt](https://github.com/k14s/ytt) to create and maintain reusable YAML templates. You can visit the ytt [playground](https://get-ytt.io/) to learn more about its templating features.
-      > In the above command, `ytt` can take a folder e.g. `config` or file via `-f`. See all options by running `ytt help`.
 
       ii. Install using `kapp` and pass the above K8s configuration file
       ```console
       kapp deploy -a cf -f ${TMP_DIR}/cf-for-k8s-rendered.yml -y
       ```
-      > cf-for-k8s uses [kapp](https://github.com/k14s/kapp) to manage its lifecycle. `kapp` will first show you a list of resources it plans to install on the cluster and then will attempt to install those resources. `kapp` will not exit untill all resources are installed and their status is running. See all options by running `kapp help`.
 
-   Once you run the command, it should take about 10 minutes depending on your cluster bandwidth, size. `kapp` will provide updates on pending resource creations in the cluster and will wait until all resources are created and running. Here is a sample snippet from `kapp` output:
+   Once you run the command, it should take about ~5-7 minutes depending on your cluster bandwidth, size. `kapp` will provide updates on pending resource creations in the cluster and will wait until all resources are created and running. Here is a sample snippet from `kapp` output:
```

Comment:
I'd leave it at "about 10 minutes"

Reply:
Do we instrument installation to track where the time is spent? ~10 minutes sounds a lot of time (even 5-7 minutes sounds like worth looking into).

Reply:
To clarify, we're talking about fresh install time and I think we're feeling rather proud about the relatively fast install time here (especially compared to cf for VMs).

My understanding is that it takes 4+ minutes to go from "fresh cluster" to "ready to `cf push`" largely because there are a few component dependencies (eg UAA requires the database to be up and running, log cache requires UAA, etc).

Reply:
Maybe change it to "about 10 minutes or less, depending on your cluster ..."?